src/components/Modal-custom-btn/ModalCustomBtnEmail.tsx

```diff
@@ -3,18 +3,11 @@ import { Link } from 'react-router-dom'
 import { Typography, Stack } from '@mui/material'
 import axiosInstance from '../../axios/settings'
 import IconinfoMessage from '../../icons/infoMessage.svg?react'
-import InfoConfirmationModal from './ModalCustomWindow'
 import { Button } from '../UI/Button'
-import {
-  stEmailP2,
-  stEmailSpan,
-  stIconEmail,
-  stIconLinkEmail,
-  stLinkEmail,
-  stLinkEmailList,
-  stLinkEmailP
-} from './style'
+
 import { useSnackbar } from '../../hooks/useSnackbar'
+import { btn, emailP2, iconLinkEmail, linkEmailList } from './style'
+import Modalka from './ModalCustomWindow'
```

Comment:
that is wrong name for component, use english words

Reply:
done